cmd/notify-mailer/main.go

```diff
@@ -142,7 +217,12 @@ func main() {
 
 	toBody, err := ioutil.ReadFile(*toFile)
 	cmd.FailOnError(err, fmt.Sprintf("Reading %s", *toFile))
-	destinations := strings.Split(string(toBody), "\n")
+
+	var contacts []contact
+	err = json.Unmarshal(toBody, &contacts)
+	cmd.FailOnError(err, fmt.Sprintf("Unmarshaling %s", *toFile))
+	destinations, err := resolveDestinations(contacts, dbMap)
```

Comment:
This seems to break check-pointing (i.e. if emails are added/removed/changed between runs).